src/app/todo-footer/todo-footer.component.html

```diff
@@ -0,0 +1,3 @@
+
+<h3> Items completados: {{itemsCompletados}}</h3>
```

Comment:
aca en vez de bindear con una variable, yo binderaria con una funcion

{{itemsCompletados()}}

y despues la funciuon hace algo como 

itemsCompletados() {
return this.items.filter(item => item.isCompletado).reduce((sum, each) => sum + each, 0)
}

Reply:
la funcion se va a ejeciutat de nuevo sola acada vez que le pases los items a este componente

Reply:
Profe, ahí hice el cambio; no me funcionó el reduce(), usé otro método que encontré en internet.